packages/js/src/components/SidebarCollapsible.js

```diff
@@ -32,6 +32,7 @@ const SidebarCollapsible = ( props ) => {
 				onClick={ handleClick }
 				className="components-button components-panel__body-toggle"
 				type="button"
+				id={ props.buttonId }
```

Comment:
Minor thing: on line 17 the props are spread, but you didn't add the `buttonId` there, but are using it directly here.